public/locales/de/invitationModal.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Invite users to {{name}}": "Benutzer*innen zu {{name}} einladen",
+  "invitedUser_one": "{{ count }} Benutzer*in wurde eingeladen und muss die Einladung annehmen",
+  "invitedUser_other": "{{ count }} Benutzer*innen wurden eingeladen und müssen die Einladung annehmen",
+  "Error while trying to fetch users. ": "Error beim Abruf der User.",
```

Comment:
this is a really _meh_ translation... 😕

Reply:
Also: Why do we have a period here? But no periods at the end of sentences above?